server/sandbox_run_linux.go

```diff
@@ -680,12 +690,16 @@ func (s *Server) runPodSandbox(ctx context.Context, req *pb.RunPodSandboxRequest
 	return resp, nil
 }
 
-func setupShm(podSandboxRunDir, mountLabel string) (shmPath string, err error) {
+func setupShm(podSandboxRunDir, mountLabel string, shmSize int) (shmPath string, err error) {
+	if shmSize <= 0 {
+		return "", fmt.Errorf("ShmSize must be greater than 0")
+	}
+
 	shmPath = filepath.Join(podSandboxRunDir, "shm")
 	if err := os.Mkdir(shmPath, 0700); err != nil {
 		return "", err
 	}
-	shmOptions := "mode=1777,size=" + strconv.Itoa(libsandbox.DefaultShmSize)
+	shmOptions := "mode=1777,size=" + strconv.Itoa(shmSize)
```

Comment:
this would fail if a user put `64M` right? I am trying to find how we handle file size sanitization. @saschagrunert @mrunalp do y'all remember

Reply:
But once someone remembers that package for me, we should use that, rather than itoa/atoi, to allow for human readable byte limits

Reply:
We're using `github.com/docker/go-units.FromHumanSize()` elsewhere.

Reply:
@nalind I notice that `github.com/docker/go-units.FromHumanSize()` will parse all `1K`, `1KB` and `1KiB` into `1000`, while `github.com/docker/go-units.RAMInBytes()` parse them into 1024. 
As far as I know `1K` and `1KB` should be parsed as `1000` bytes, and `1KiB` should be parsed as `1024` bytes. Am I understanding correctly? Or I can just use `github.com/docker/go-units.FromHumanSize()` anyway?

Reply:
@nalind I found another package `k8s.io/apimachinery/pkg/api/resource.ParseQuantity()` which more flexible, should we use this one?

Reply:
I am using `resource.ParseQuantity` in machine-config-operator, https://github.com/openshift/machine-config-operator/blob/master/pkg/controller/kubelet-config/helpers.go#L144

Reply:
Since that package is maintained by upstream k8s project, your parsing results will always be consistent with them. So I would recommend using that package.

Reply:
thanks @harche, I agree, let's go with ParseQuantity